packages/next/src/server-build.ts

```diff
@@ -313,7 +313,7 @@ export async function serverBuild({
 
     const useBundledServer =
       semver.gte(nextVersion, BUNDLED_SERVER_NEXT_VERSION) &&
-      process.env.VERCEL_NEXT_BUNDLED_SERVER;
+      process.env.VERCEL_NEXT_BUNDLED_SERVER === '1';
```

Comment:
Or you can coerce it into a boolean :)

```suggestion
      ~~process.env.VERCEL_NEXT_BUNDLED_SERVER;
```

Reply:
Too clever / unreadable IMO. I have no idea what that code does just by looking at it.

Reply:
I didn't even know `~` is an operator lol